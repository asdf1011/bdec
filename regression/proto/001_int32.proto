message Integer {
    required int32 value = 1;
}
