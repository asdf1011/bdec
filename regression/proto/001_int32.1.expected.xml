<Integer>
    <value>2147483647</value>
</Integer>
